src/pages/careers/front-end-developer.pug

```diff
@@ -0,0 +1,92 @@
+extends ../../layouts/content.pug
+
+block prepend vars
+  -var pageTitle = "Front-end Developer Position | Leaf"
+  -var metaUrl = "https://weareleaf.com/careers/front-end-developer"
+  -var metaTitle = "Front-end Developer Position"
+  -var metaDescription = "We're looking for a brilliant front-end developer to build world-class web applications at Leaf. Sound like your thing? Read on, and get in touch."
+
+block pageComponents
+
+  +banner({})
+
+  .banner-spacer
+
+  +hero({
+    heroHeading: metaTitle,
+    heroSubheading: metaDescription,
+  })
+
+  .section
+    .container
+      figure
+        img(src="/assets/images/careers/image-1.jpg" alt="Mike, hard at work on his Macbook")
+
+      .grid.grid--two-thirds-reverse
+        .grid__item
+          +summary({
+            summaryHeading: "Who are we?",
+            summaryText: "Leaf are a remote agency based in the UK. We specialise in complex information capture, helping organisations excel online by making it easy for their users to provide them with high quality information."
+          })
+
+        .grid__item
+          h4 About the role
+          p We're working hard to build a strong core of expertise around complex information capture on the web. To get there, we need help from a brilliant front-end developer who loves to build fast, accessible, progressively enhanced web applications that deliver tremendous value.
+
+          p In this role you'll be working remotely and choosing your own hours, but within that flexibility you'll be expected to deliver features and improvements in an efficient, timely manner. We can't promise every day will be radically different and exciting, but we also know not everyone wants that. You'll spend most of your time problem solving, writing code, and shipping value.
```

Comment:
Putting myself in job-hunting shoes, the last two sentences are a real-turn off for me. Could we flip the negative and outline who it's ideal for instead?

> The role is ideal for someone who likes stability and routine, as you'll spend most of your time problem solving, writing code, and shipping value.